### ISSUE TITLE
Missed taint sink caused by aliasing due to shallow list copy

### ISSUE THREAD

**Issue by draftyfrog** [facebook/pyre-check#923]
Please consider the following code where we create a list, give it an alias by copying the reference and adding tainted data using the alias:
```python
    original: list[str] = []
    copy: list[str] = []
    copy = original
    copy.append(source())
    sink(original[0])  # Not found by pysa
    sink(copy[0])  # Found by pysa 
```
My `.pysa` config file looks like this
```
def source() -> TaintSource[TestSource]: ...
def sink(param: TaintSink[TestSink]): ...
def list.append(self, __m: TaintInTaintOut[Updates[self]]): ...
```

As both names `original` and `copy` point to the same list, both sinks should be reported by Pysa but it only reports the sink when we use the new alias. Do I miss some configuration?

**Comment by arthaud** [comment]
Hi @draftyfrog,

Unfortunately, this is a known problem in our current analysis.
We do not have strong indications that it is leading to many missed issues, so we are not currently prioritizing fixing this.
